pyxem/signals/crystallographic_map.py

```diff
@@ -170,10 +166,10 @@ class CrystallographicMap(BaseSignal):
         Method used to obtain crystallographic mapping results, may be
         'template_matching' or 'vector_matching'.
     """
+    _signal_dimension = 1
     _signal_type = "crystallographic_map"
 
     def __init__(self, *args, **kwargs):
-        self, args, kwargs = push_metadata_through(self, *args, **kwargs)
```

Comment:
Could you briefly explain why this is no longer required, please? I'm sure @pc494 had reasons for introducing it but I can't recall them right now.

Reply:
I removed this, because the metadata dictionary can be passed as an argument of `__init__`, this is more explicit and shorter. Along the same lines, I find that the instantiation of a signal with another signal is not clean pattern because not all the signal attributes are passed to the other signal and it is not clear what we should expect from this.

Reply:
Yes, I think we agree on this and what we had was always just supposed to be a dirty fix until this signal registration mechanism became available - very grateful for that!